Documentation/Guides/Error Handling.md

```diff
@@ -0,0 +1,48 @@
+### Basic Error Handling
+
+As a general rule, networking is hard. A lot can go wrong in the process of communicating between an application and a backend. One of the goals of Hyperspace is to make the many different failure reasons easy to understand and easy to react to. In this section, we'll describe the steps needed to do the bare minimum to effectively handle errors.
+
+In the vast majority of scenarios, your app will interface with Hyperspace by creating an object that conforms to the `Request` protocol and handing it to a `BackendService`. At this point, the `BackendService` will unwrap the underlying `URLRequest` and forward it on to an object called a `TransportService`, which in turn uses a `NetworkSession` to execute the request itself. Each of the layers in this process play a distinct role in the error handling process.
```

Comment:
Minor update to wording since I don't think we have the `Request` protocol anymore.
```suggestion
In the vast majority of scenarios, your app will interface with Hyperspace by creating a `Request` struct and handing it to a `BackendService`. At this point, the `BackendService` will unwrap the underlying `URLRequest` and forward it on to an object called a `TransportService`, which in turn uses a `NetworkSession` to execute the request itself. Each of the layers in this process play a distinct role in the error handling process.
```

Reply:
Fixed